packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -54,15 +60,17 @@ export class NavigatorWrapper extends React.Component<Props> {
   }
 
   render() {
+    const { appLocked } = this.props
     return (
       <View style={styles.container}>
         <AppContainer
           ref={this.setNavigator}
           onNavigationStateChange={navigationStateChange}
           {...getPersistenceFunctions()}
         />
+
         <View style={styles.floating}>
-          <BackupPrompt />
+          {!appLocked && <BackupPrompt />}
           <AlertBanner />
```

Comment:
Should alert banner be conditional too?

Reply:
We need `<AlertBanner />` to show user `Incorrect PIN` if that's the case.